### ISSUE TITLE
Service Worker -> Web app communication: Svelte store.

### ISSUE THREAD

**Issue by DanielBanobreDopico** [Mooli-me/PoC#2]
A web app with a Service Worker that write some data (datetime, increasing counter, nevermind) over and over to a Svelte store, and this data is updated in the frontend.
*Reference: https://svelte.dev/docs#svelte_store*

**Comment by DanielBanobreDopico** [comment]
After some tests, I found there is not a simple way for import a Svelte store from a simple .js script. It's becouse the service worker script is nor a Svelte module nor a js library loaded by Svelte module then Svelte don't compile it and Svelte falicities don't work.

Searching for solutions,  I found that the new SvelteKit project [supports service workers](https://kit.svelte.dev/docs#service-workers) and dinamically build those server side.

I'm going to get a try.
